.circleci/ci/it-tests.js

```diff
@@ -49,7 +49,6 @@ try {
     // Start CQ
     ci.sh(`./qp.sh -v start --id author --runmode author --port 4502 --qs-jar /home/circleci/cq/author/cq-quickstart.jar \
             --bundle org.apache.sling:org.apache.sling.junit.core:1.0.23:jar \
-            --install-file ${config.modules['core-forms-components-it-tests-apps', 'core-forms-components-it-tests-content'].path}/src/main/resources/com.adobe.granite.toggle.impl.dev-1.1.2.jar \
```

Comment:
how are these packages getting installed now?

Reply:
We do not want granite toggle bundle to be installed on the server since it is internal to Adobe. Moreover, since we now use prerelease tag with the QS we do not need this. This was added when there was no support of prerelease QS in this repo.

Reply:
added another argument to existing command to install core-forms-components-it-tests-apps.
currently core-forms-components-it-tests-content is being installed using a separate argument.